### DISCUSSION TITLE
Catch all errors and show Notification

### DISCUSSION THREAD

**Discussion by holymp2006** [filamentphp/filament#10938] [Help]
In production, an error modal appears whenever an error occurs. I'm wondering if there is a way to globally intercept or catch all errors in production—perhaps from `Handler.php` or a hook—`report()` the errors, and show the user a simple, unobtrusive error notification (`Notification::send()`).

**Comment by leandrocfe** [comment]
Maybe using something like this?

https://filamentphp.com/docs/3.x/forms/validation/#sending-validation-notifications

**Reply by holymp2006** [reply]
Exactly! Something like validation notifications, but for all errors.

So, instead of something like this:
![error-modal](https://github.com/filamentphp/filament/assets/13824510/c967196c-ae73-4412-93b5-9bad943452af)

I want to replace with this:
![error-notification](https://github.com/filamentphp/filament/assets/13824510/33eaffe6-38b2-4761-b904-100545f16e45)

**Comment by sh4msi** [comment]
Is there a solution to this issue?

@leandrocfe @holymp2006

**Comment by arxjei** [comment]
maybe you can try this. https://laravel.com/docs/12.x/errors with this https://filamentphp.com/docs/3.x/forms/validation#sending-validation-notifications
